packages/plugins/users-permissions/admin/src/pages/Roles/EditPage/index.js

```diff
@@ -70,16 +84,17 @@ const EditPage = () => {
         onSubmit={handleCreateRoleSubmit}
         validationSchema={schema}
       >
-        {({ handleSubmit }) => (
-          <form onSubmit={handleSubmit}>
+        {({ handleSubmit, values, handleBlur, handleChange, errors }) => (
+          <>
             <HeaderLayout
               id="title"
               primaryAction={
-                !isLoading && (
+                !isLoadingPlugins && (
                   <Button
                     disabled={role.code === 'strapi-super-admin'}
                     onClick={handleSubmit}
```

Comment:
You don't need to pass the onClick event here you just need to add a `type="submit"` attr.